code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -0,0 +1,518 @@
+#define CREDIT_REWARD_BASE 100
```

Comment:
Still unsure about making them defines and not vars.

Reply:
I suppose admins will want to VV contractor stuff at some point, so why not.